Source/PLCrashReporter.m

```diff
@@ -920,8 +931,8 @@ - (id) initWithBundle: (NSBundle *) bundle configuration: (PLCrashReporterConfig
         PLCR_LOG("Warning -- bundle version unavailable");
         bundleVersion = @"";
     }
-    
-    return [self initWithApplicationIdentifier: bundleIdentifier appVersion: bundleVersion appMarketingVersion:bundleMarketingVersion configuration: configuration];
+
+    return [self initWithApplicationIdentifier: bundleIdentifier appVersion: bundleVersion appMarketingVersion:bundleMarketingVersion basePath:basePath configuration: configuration];
```

Comment:
```suggestion
    return [self initWithApplicationIdentifier: bundleIdentifier appVersion: bundleVersion appMarketingVersion:bundleMarketingVersion basePath: basePath configuration: configuration];
```

Reply:
Please help change it to comply with your standards. Thanks!
I learned at from this PR.

Reply:
Got you. Is that okay if I commit to to a branch in your repository?

Reply:
Yes, it is ok. Please do it.Thanks!